packages/teams-js/src/public/media.ts

```diff
@@ -683,6 +683,9 @@ export namespace media {
   }
 
   /**
+   * @deprecated
+   * As of 2.0.1, please use {@link barCode.scanBarCode barCode.scanBarCode(config?: BarCodeConfig): Promise\<string\>} instead.
```

Comment:
@erinha Should we be using 2.0.1 here?

Reply:
Good catch. No, we should use 2.1.0 for this. A good way to determine what the next version will be for deprecated tag purposes is by locally running `yarn beachball bump` from the root of the monorepo and seeing what the version is in packages/teams-js/package.json as a result. 

NOTE: Make sure you do that without any other changes in your local repo because it will delete a bunch of change files and modify the packages/teams-js/package.json and you'll want to undo those changes after you run the command.

Reply:
Thanks for catching this Jay and for the info Erin!